matter/wifi/rs911x/rsi_if.c

```diff
@@ -435,7 +456,7 @@ void wfx_rsi_task(void *arg) {
           hasNotifiedIPV4 = false;
         }
 #endif /* CHIP_DEVICE_CONFIG_ENABLE_IPV4 */
-        if ((ip6_addr_ispreferred(netif_ip6_addr_state(sta_netif, INDEX))) &&
+        if ((ip6_addr_ispreferred(netif_ip6_addr_state(sta_netif, 0))) &&
```

Comment:
Why `0` ? is the index `0` is always valid ? needs at least a comment explaining why this is done

Reply:
added comment , here we are evaluating block 0 of assigned IPv6 address for it's validity.